### DISCUSSION TITLE
Two Questions on P-F

### DISCUSSION THREAD

**Discussion by jsevic** [idaholab/moose#17979] [Q&A Modules: Phase field] [answered]
Hello,

Please see below two questions on P-F.

Thank you.

John Sevic

Q1.
In the file direct_order4_test.i located at (modules/phase_field/test/tests/PolynomialFreeEnergy/direct_order4_test.i) there are two references, Brown1980 and Balluffi1978, is there a master reference section that precisely defines these references.

Q2.
Please refer to the attached screen captures returned when I run direct_order4_test.i Any suggestions? I should probably update my MOOSE, if that helps.

![Screen Shot 2021-05-31 at 5 19 39 PM](https://user-images.githubusercontent.com/4711942/120250878-7b70a580-c234-11eb-93c0-9be061007765.png)
![Screen Shot 2021-05-31 at 5 19 26 PM](https://user-images.githubusercontent.com/4711942/120250881-7f042c80-c234-11eb-94d1-50f51af77930.png)

**Comment by laagesen** [comment, marked as answer]
Hi John,

The references for phase-field documentation are listed at projects/moose/modules/phase_field/doc/content/bib/phase_field.bib. Unfortunately I don't see either of the listed references there and it looks like this input file pre-dated the documentation system anyways. The person who added this input file no longer works at INL so I'm afraid it might not be that easy to track down this reference.

The error you are getting indicates you need to update your PetSC library and likely MOOSE as well.
